### ISSUE TITLE
[L] UI - View all Available Park Name Info

### ISSUE THREAD

**Issue by ashley-dhillon** [bcgov/parks-data-register#28]
### **Updated Acceptance Criteria**

Design matches Figma designs

User is able to:

**View all details associated with a name**
- Type
- Status
- Last updated
- ~~_Notes_~~ (will be tackled via separate ticket/issue)
- ~~_Search terms_~~ (will be tackled via separate ticket/issue)
- ~~_Recent changes~~ (change log needs to be designed still so add title and content will be added later)_
- ~~_List of associated sites~~ (data to be hooked up once sites are added to data model in the future)_

**Page Title**
made up of the ID number and the Legal Name, if legal name is too long, wrap to second line

**Edit Record Button**
on click goes to (1.1.1) edit record - choose edit type

**Type, Status and last updated** 
Data directly associated with the protected area ID
None of these data points are editable

**Type** - is indicated by the type user selects during the protected area ID creation process (future)
**Status** - will be generated based on users actions:
- After creating a new protected area status will be Pending (future)
- After adding a legal name to a pending protected area, status will change to Established
- After repealing the park, status will change to Repealed
**Updated date** is the most recent date of edits or changes. Display in this format:
Month DD, YYYY e.g., December 27, 2021

**View Legal name details** 
- Effective date
- Current Legal name
- Display name
- Phonetic name
- Audio URL

Will display the current legal name and all the fields associated with a name. Those are:
**Effective date** - the date the name change took effect, display in same format as in #3 above
**Current legal name** - the name as it appear on the legal paperwork
**Display name** - the name to be used on public-facing sites. Usually the same as the legal name except in a few cases.
**Phonetic name** - uses standard English alphabet
**Audio link** - the URL where the audio clip is stored

**View historical name details**
- Effective date
- Historical Legal name
- Display name
- Phonetic name
- Audio URL

Will display historical name and all the fields associated with that name – which are the same fields as listed above with current name. There can be multiple historical names.

Historical names come from:
The original dataset
Added automatically as part of the name change flow. When a new name is created, the old name and associated fields will become a historical record.

![Image](https://github.com/bcgov/parks-data-register/assets/92062502/63a1eb01-35a5-4b49-8d46-abf0c570ab3f)

**Repealed status**
status will display as "Repealed" for repealed protected areas

**Repealed date**
Date will be added by user during repeal flow, can be edited by user in edit flow

**Empty Fields**
Where there are fields with no data, display None as shown.

![Image](https://github.com/bcgov/parks-data-register/assets/92062502/0df55b63-fada-4c60-ba13-94b9ba29babc)

Figma files have been reviewed by all team members:

[View page with a lot of data](https://www.figma.com/file/qdjegD6YHWZPlyFT8dGXvk/Park-Name-Data-Register?type=design&node-id=543%3A2329&mode=design&t=w3cPISuB25H6NaLQ-1)
[View page with only a little data](https://www.figma.com/file/qdjegD6YHWZPlyFT8dGXvk/Park-Name-Data-Register?type=design&node-id=908%3A4908&mode=design&t=w3cPISuB25H6NaLQ-1)
[View page for a repealed park]
(https://www.figma.com/file/qdjegD6YHWZPlyFT8dGXvk/Park-Name-Data-Register?type=design&node-id=908%3A5377&mode=design&t=Jb0qJNQucP7YZBs4-1)

Notes:
- Site are not included with this ticket will be added in the future
- Notes field not included in this release - needs to be further refined
- Recent changes section not included in this release - needs to pull from Change Log

*** Updated 2023-10-17 - DT ***
- Effective date is a field that is manually set via the edit workflow. This is mapped to the field in the record called effectiveDate.
- Type is hard coded to Protected Area. This is current state for MVP.
- Status -> status
- Last updated -> updateDate
- Legal name -> legalName
- Display name -> displayName
- Phonetic name -> phoneticName
- Audio link -> audioClip
- Search terms is currently a mega string. This will be passed to Mark's backend work https://github.com/bcgov/parks-data-register/issues/134.
- Associated sites does not need to be completed. It is not MVP.
- Recent changes does not need to be completed. It is not MVP.
- Notes field requires more refinement. Moving to blocked.

**Comment by danieltruong** [comment]
Effective date is a field that is manually set via the edit workflow. This is mapped to the field in the record called effectiveDate.

**Comment by danieltruong** [comment]
Type is hard coded to Protected Area. This is current state for MVP.

**Comment by danieltruong** [comment]
Status -> status
Last updated -> updateDate
Legal name -> legalName
Display name -> displayName
Phonetic name -> phoneticName
Audio link -> audioClip

**Comment by danieltruong** [comment]
Search terms is currently a mega string. This will be passed to Mark's backend work [#134](https://github.com/bcgov/parks-data-register/issues/134).

**Comment by danieltruong** [comment]
Associated sites does not need to be completed. It is not MVP.

**Comment by danieltruong** [comment]
Recent changes does not need to be completed. It is not MVP.

**Comment by danieltruong** [comment]
Notes field requires more refinement. Moving to blocked.

**Comment by JLWade** [comment]
@danieltruong since we decided to keep notes field hardcoded and empty for now (ie. just a space in UI held for Notes) can we move this one out of blocked?

**Comment by danieltruong** [comment]
@JLWade Should be fine to move back into to do. I'll be working on hooking up the API to the frontend with @meredithom  either tomorrow and Wednesday.

**Comment by danieltruong** [comment]
blocked by design changes

**Comment by manuji** [comment]
Verified that the functionality has been implemented but comprehensive testing will be covered when https://github.com/bcgov/parks-data-register/issues/294 is ready for QA
